protecode/model.py

```diff
@@ -32,6 +34,12 @@ class ProcessingStatus(Enum):
     FAILED = 'F'
 
 
+@dataclasses.dataclass(frozen=True)
+class UploadElement:
```

Comment:
I think this does not reasonably belong to protecode's model module. Might be a better fit for either `gci.componentmodel`, or our local `cnudie` package. The name is also too strongly influenced by the use-case for which it was added. It is in fact a two-tuple of a resource and its resource. Why not simply call it ComponentResource.

for the time being, at least mv it to protecode.scanning_util